recipes/serapeum_utils/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "serapeum_utils" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/serapeum_utils/archive/{{ version }}.tar.gz
+  sha256: 657d84d21ffaba7d785d39ab296d7130618c9179d6db5bed831a3728ca379403
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.9,<3.11
+    - pip >=22.3.1
+    - poetry >=1.2.2
```

```suggestion
    - poetry-core >=1.0.0
```
Ref: https://github.com/Serapieum-of-alex/serapeum_utils/blob/6202736a1da9567f2dd53bb7adfc77bb80bbd0f6/pyproject.toml#L79